framework/wazuh/agent.py

```diff
@@ -624,10 +624,11 @@ def create_group(group_id):
         raise WazuhError(1711, extra_message=group_id)
 
     # Create group in /etc/shared
-    group_def_path = path.join(common.shared_path, 'agent-template.conf')
+    agent_conf_template = path.join(common.shared_path, 'agent-template.conf')
```

Comment:
Should we have this definition in common.py with the rest of the paths?

Reply:
It is the only reference to that path in the code. I would only add it if we need it again.